<!DOCTYPE HTML>

<html>
	<head>
		<title>Mostafa The Bioinformatics Portfolio</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper" class="fade-in">

				<!-- Intro -->
				<div id="intro">
					<h1>Bioinformatics portfolio
						Mostafa Torbati<br /></h1>
					<p>Bioinformatician skilled in Bioinformatician, Single Cell RNA-Seq, Analysis Omics Data Analysis & Biological Researches <a href="https://github.com/mostafa-ti/Master_thesis_bioinformatics">@GitHub project</a></p>
					<ul class="actions">
						<li><a href="#header" class="button icon solid solo fa-arrow-down scrolly">Continue</a></li>
					</ul>
				</div>

				<!-- Header -->
				
				<head>
					<style>
						/* CSS for the header */
						#header {
							background-color: #YourBackgroundColor; /* Replace #YourBackgroundColor with your desired background color */
							text-align: left; /* Aligns the text to the left (horizontal) */
							padding: 20px; /* Adds padding to provide spacing around the text */
						}
				
						#header .logo {
							display: inline; /* Makes the text display horizontally */
						}
					</style>
				</head>

				<!-- Nav -->
					<nav id="nav">
						<ul class="links">
							<li class="active"><a href="index.html">Investigation of age-related changes in neuroblast populations using RNA velocity</a></li>
						
						</ul>
						<ul class="icons">
							<li><a href="https://www.linkedin.com/in/mostafatorbati/" class="icon brands alt fa-linkedin"><span class="label">Linkedin</span></a></li>
									<li><a href="https://github.com/dashboard" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">

						<!-- Featured Post -->
							<article class="post featured">
								<header class="major">
									<h2><a href="#">POV of the Project</a></h2>
									<p class="justified-text">This work aimed to understand the aging signature
										on the neurogenesis process in the brain. For this 
										purpose, three regions of mice brain, the Dentate Gyrus 
										(DG), Subventricular zone (SVZ), and Olfactory Bulb (OB), 
										in three age sample groups young (3 months), adults (14 months), and aged (24 months) 
										were compared. This project provided an RNA velocity map of neurogenic niches of mice, 
										together with substantial new knowledge about the biology of newly born neuroblasts and 
										how they are affected by aging. Overall, our data advanced our understanding of the aging 
										signature on the neurogenesis process in the brain.</p>
								</header>
								<a href="#" class="image main"><img src="images/rna_seq.png" alt="" /></a>
								<ul class="actions special">
									<li><a href="https://github.com/mostafa-ti/Master_thesis_bioinformatics" class="button large">View Project</a></li>
								</ul>
							</article>


						<!-- Posts -->
						<style>
							.image {
								width: 100%; /* Set the width of each image to 50% */
								height: auto; /* Maintain aspect ratio */
							}
						</style>
							<section class="posts">
								<article>
									<header>
										<h2><a href="#">Adult Neurogenesis</a></h2>
									</header>
									<div class="image-block"><p>
										<br>
										
									</p>
										<img src="images/neurogenesis.png" alt="images/neurogenesis.png" class="image">
									</div>
									<p>
										<br>
									</p>


									<p> Adult neurogenesis is the process of generating new neurons from neural stem cells (NSCs) in the adult mammalian brain, which occurs in two neurogenic niches; the subventricular zone (SVZ) of the lateral ventricles and the subgranular zone (SGZ) of the hippocampal dental gyrus (DG). In rodents, neurogenesis continues throughout life, but this process decreases with age, which may result in cognitive decline. However, it is still poorly understood how aging affects the cell differentiation process throughout neurogenesis. </p>
									
								</article>
								<article>
									<header>
										
										<h2><a href="#">Exploring The Continuum of Neurogenesis</a></h2>
									</header>
									<div class="image-block">
										<img src="images/continuum.png"  alt="images/continuum.png" class="image">
									</div>
									<p>In our study, we used scRNA-seq transcriptomics and RNA velocity analysis to investigate neurogenesis across age groups. We compared young, adult, and aged neuroblasts, uncovering age-related transcription changes. Notably, we found a reverse differentiation pattern in aged mice, with neurons reverting to neuroblasts. Young and adult groups shared a similar neuroblast-to-neuron trajectory. Furthermore, older neuroblast clusters showed increased inflammatory gene expression. These findings deepen our understanding of how aging affects neuroblast development, suggesting a potential cell de-differentiation process in aged SVZ.</p>
									
								</article>
								<article>
									<header class="major">
									
										<h2><a href="#">Experiment Design</a></h2>
									</header>
									<p>
										<br>
										
									</p>
									<a href="#" class="image fit"><img src="images/Experiment_design.png" alt="" /></a>
									<p>Samples were sourced from transgenic mice with EGFP under the DCX promoter. Neuroblasts from DG, SVZ, and OB regions in mice of various ages (3 months, 18 months, 24 months) were dissected, enzymatically processed, and sorted using FACS to investigate transcriptional changes during aging.</p>
								</article>
								<style>
									/* CSS for styling the header */
									header {
										background-color: white; /* Background color for the header */
										color: black; /* Text color for the header */
										padding: 10px; /* Padding around the content within the header */
										text-align: center; /* Center the content horizontally */
									}
							
									/* CSS for styling the logo */
									.logo {
										width: 100px; /* Set the width of the logo */
										height: auto; /* Maintain the logo's aspect ratio */
									}
								</style>
								<article>

									<header>
										<img src="images/10x_Genomics_logo.svg.png" alt="Logo" class="logo">
									
								</header>
								<a href="#" class="image fit"><img src="images/10X.mp4" alt="" /></a>
								<p>Cells were sorted and processed using a 10X genomics droplet-based protocol. Unique Molecular Identifiers (UMIs) were used to barcode and tag each cell. The tagged cells underwent Illumina sequencing, which was performed twice to enhance depth. The raw data was obtained in the form of Binary Base Call (BCL) files from the sequencer.</p>
							
								</article>
								<article>
									<header>
										<h2><a href="#">Bioinformatics Pipelines</a></h2>
									</header>
									<a href="#" class="image fit"><img src="images/project_flowchart-3.png" alt="" /></a>
									<ul class="actions special">
										<li><a href="https://github.com/mostafa-ti/Master_thesis_bioinformatics#flowchart" class="button">Follow Here</a></li>
									</ul>
								</article>
								<article>
									<header>
										<h2><a href="#">CellRanger pipeline:<br />Raw data Conversion</a></h2>
									</header>
									<a href="#" class="image fit"><img src="images/cell_ranger.png" alt="" /></a>
									<p>The Cellranger is provided by 10X genomics and includes analysis pipelines that process single-cell RNA-seq output to prepare the data for downstream analysis. All the steps were performed using the Cell Ranger v6.0 pipelines as described<a href="https://github.com/mostafa-ti/Master_thesis_bioinformatics#converting-raw-data-to-fastq-files" style="color: blue; text-decoration: none;"> IN THIS LINK.</a>
										<br />
										<br />
										<br />
										<style>
											.actions.special .button {
											  text-decoration: none;
											  display: inline-block;
											}
										  
											.actions.special .button img {
											  max-width:95px; /* Adjust the width as needed */
											  max-height: 75px; /* Adjust the height as needed */
											  border: none !important;; /* Remove the border */
											}
										  </style>
										  
										  <ul class="actions special"> 
											<li>
											  <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger" class="button">
												<img src="images/10x_Genomics_logo.svg.png" alt="10x Genomics Logo">
											  </a>
											</li>
										  </ul>
								</article>
								<article>
									<header>
										
										<h2><a href="#">Bioinformatic Analyses</a></h2>
									</header>
									<a href="#" class="image fit"><img src="images/bioinformatics_analysis.drawio.png" alt="" /></a>
									<p><a href="https://scanpy.readthedocs.io/en/stable/" style="color: blue; text-decoration: none;"> Scanpy</a> provides the foundation for single-cell RNA sequencing (scRNA-seq) data analysis. <a href="https://velocyto.org/velocyto.py/tutorial/analysis.html#velocyto-loom" style="color: blue; text-decoration: none;"> Velocyto</a> complements Scanpy by quantifying RNA splicing dynamics, a key component for velocity analysis.<a href="https://scvelo.readthedocs.io" style="color: blue; text-decoration: none;"> ScVelo</a>, an extension of Scanpy, specializes in RNA velocity analysis, enabling us to study gene expression changes over time, enriching single-cell data analysis with a temporal dimension. Collectively, by these tools we explored gene expression dynamics in neuroblast populations.</p>
									<ul class="actions special">
										<li><a href="https://github.com/mostafa-ti/Master_thesis_bioinformatics/blob/master/scanpy.ipynb" class="button">Code Availability</a></li>
									</ul>
								</article>
								<article>
									<header>
										<h2><a href="#">Quality Control</a></h2>
										<br />
									</header>
									<a href="#" class="image fit"><img src="images/QC.png" alt="" /></a>
									<br />
									<p>Quality control procedures were implemented to filter out low-quality cells and putative doublets. Quality control metrics for each cell, such as the number of detected genes, the total count, and percentages of mitochondrial genes, were plotted. <br />Following the quality control and normalization steps, the dataset was refined to a total of 4,635 cells, with 2,487 highly variable genes remaining.</p>
									
								</article>
								<article>
									<header>
										<h2><a href="#">Leiden Clustering</a></h2>
									</header>
									<a href="#" class="image fit"><img src="images/umap_leiden.png" alt="" /></a>
									<p>Transcriptionally similar cells were clustered based on an established Leiden algorithm, resulting in 14 divisible clusters. Subsequently, a UMAP plot was generated with a resolution set at 0.5, showcasing data from the subventricular, dentate gyrus, and olfactory bulb zones across different age groups, including young (3 months), adults (14 months), and aged (24 months) mouse brain.</p>
									
								</article>
								<article>
									<header>
										<h2><a href="#">Marker Genes</a></h2>
									</header>
									<a href="#" class="image fit"><img src="images/stacked_violin_all_data_violinMarkerGenes.jpg" alt="" /></a>
									<div class="content">
										<p class="intro-paragraph">
											To annotate cell clusters, we selected the best available marker genes corresponding to various cell types from the literature:
										</p>
										<ol class="marker-genes">
											<li><strong>Tfap2c</strong> for DG neural stem cells</li>
											<li><strong>Ascl1</strong> for Transient Amplifying Progenitors (TAPs)</li>
											<li><strong>DCX</strong> and <strong>Igfbpl1</strong> for neuroblasts</li>
											<li><strong>Aqp4</strong> and <strong>Gfap</strong> for Astrocyte-like cells</li>
											<li><strong>Usp18</strong> for inflammatory cells</li>
											<li><strong>Pdgfra</strong> for Oligodendrocyte Progenitor Cells (OPCs)</li>
											<li><strong>Mog</strong> for mature oligodendrocytes</li>
											<li><strong>Cx3cr1</strong> for Microglial cells</li>
											<li><strong>Syt1 (Synaptotagmin 1)</strong> for Neurons</li>
										</ol>
								</article>
								<article>
									<header>
										<h2><a href="#">Cell Cluster Annotation</a></h2>
									</header>
									<a href="#" class="image fit"><img src="images/annotated.png" alt="" /></a>
									<p>Cluster annotation based on candidate marker genes.<br />All data preprocessing and annotation were conducted on a single combined dataset known as "anndata." This comprehensive dataset was created by merging information from all three age groups and distinct regions. In simpler terms, the "anndata" encompasses nine individual datasets, namely DG3, DG14, DG24, SVZ3, SVZ14, SVZ24, OB3, OB14, and OB24.<br /> Note:<br />3 for 3 Month samples, 14 for 14 month samples and 24 for 24 month samples.</p>
								</article>
								<article>
									<header>
										<h2><a href="#">Subsetting the Data	</a></h2>
									</header>
									<a href="#" class="image fit"><img src="images/subsetting.png" alt="" /></a>
									<p>Each of these datasets was organized as a Pandas data frame. This organization allowed us to perform subsequent analyses by extracting specific portions of the data, which could include data from a single region or a combination of different regions and age groups, enabling us to focus our analysis on the specific data of interest.</p>
								</article>
								<article>
									<header>
										<h2><a href="#">RNA Velocity Analysis<br />"ScVelo"</a></h2>
									</header>
									<br />
									<p>RNA Velocity serves as a pivotal tool in single-cell genomics, offering a unique perspective into the dynamic behavior of individual cells. By measuring the rate of gene expression changes over time, it acts as a compass, guiding researchers in mapping out cellular trajectories, differentiations, and state transitions. This innovative approach provides invaluable insights into complex biological systems, shedding light on the ever-changing cellular landscape and contributing to a deeper understanding of critical biological processes and disease mechanisms.<br />In this project, RNA Velocity in the neurogenic lineage was computed using ScVelo 0.2.4 (Bergen et al., 2020) with 2400 highly variable genes per cell. Moments were calculated using the full space of a pre-computed PCA (default value = 30) and 30 neighbors. Velocity was estimated using the dynamical model, and all maps were generated using the original UMAP coordinates.</p>
									
								</article>
								<article>
									<header>
										<h2><a href="#">Velocity maps of the combined data corresponding SVZ and OB</a></h2>
									</header>
									<a href="#" class="image fit" onclick="showImage('images/scvelo.png');"><img src="images/scvelo.png" alt="" /></a>
									<script>
										function showImage(imageSrc) {
											// Create a new window to display the full-size image
											window.open(imageSrc, 'Full Size Image', 'width=800, height=600');
										}
										</script>
										

									<p>In our dataset, we noticed the absence of one cluster (Astrocytes & TAPs) in the aged group, signaling a shift in aging cellular dynamics. Additionally, we observed a reverse transition from neurons to neuroblasts, with fewer instances in the young group.

										The data indicates increased dedifferentiation in adults and the aged, particularly in neurons reverting to a neuroblast state, possibly to sustain neurogenesis by generating more neurons. This reverse transition is more prominent in the aged population, suggesting that aged neurons may lose their original identity. These insights illuminate the complex cellular dynamics across age groups and their impact on neurogenesis.</p>
									<ul class="actions special">
										<li><a href="#" class="button" onclick="showImage('images/scvelo.png')">Click for high quality image</a></li>
									</ul>
								</article>
								<article>
									<header>
										<h2><a href="#">Discussion</a></h2>
									</header>
									
									<p>In our study, we made some interesting discoveries regarding the differentiation of cells in young and adult mice. We found that both age groups followed a similar path of cell differentiation, transitioning from immature neuroblasts to neurons at the transcription level. However, age seemed to have an impact on this process, suggesting a possible reverse movement from neurons back to neuroblasts, which we think could be due to de-differentiation. This finding aligns with earlier research indicating that cells can sometimes revert to an earlier state, a phenomenon observed in Alzheimer's disease and cancer research.
										Another intriguing possibility we considered is that aging may cause neurons to lose their unique characteristics. This idea is supported by previous findings that aging can lead to a loss of cell identity, ultimately resulting in neurodegeneration and disease progression.<br />
										Additionally, we noticed an increase in the expression of inflammatory genes in the neuroblast cluster of the aged SVZ-OB, which wasn't present in the younger groups. This suggests that aging may lead to increased inflammation, which can impact neurogenesis. This finding is consistent with previous studies showing that aging is associated with elevated inflammation and reduced neural stem cell proliferation.
										<br />It's important to note that while our findings are intriguing, further in vivo research is needed to confirm and expand upon these results.</p>
									
								</article>
								<article>
									<h2>GitHub Profile</h2>
									<br />
									<p>Scan the QR code or click on it to access my GitHub profile:</p>
									<br />
									<a href="https://github.com/mostafa-ti">
										<img src="images/QR.png" alt="QR Code for GitHub Profile" style="width: 300px; height: auto;">
									</a>
								</article>
								
								</article>

				<!-- Footer -->
				<style>
				  
					/* CSS for maintaining the same line alignment */
					.split.contact section {
					  display: inline-block;
					  margin: 0 20px; /* Adjust the margin as needed */
					}
				  
					.split.contact h3 {
					  font-weight: bold;
					  margin-bottom: 20px; /* Adjust as needed */
					}
				  </style>
				  
				  <footer>
					<section class="split contact">
						<div>
							<h3>Phone</h3>
							<p><a href="tel:+46763032696">(+46) 0763-032-696</a></p>
						</div>
						<div>
							<h3>Email</h3>
							<p><a href="mailto:mostafa.torbati@gmail.com">mostafa.torbati@gmail.com</a></p>
						</div>
						<div>
							<h3>Address</h3>
							<p>Lund, Sweden</p>
						</div>
						<div>
							<h3>Social</h3>
							<ul class="icons alt">
								<li>
									<a href="https://www.linkedin.com/in/mostafatorbati/" class="icon brands alt fa-linkedin">
										<span class="label">LinkedIn</span>
									</a>
								</li>
								<li>
									<a href="https://github.com/dashboard" class="icon brands alt fa-github">
										<span class="label">GitHub</span>
									</a>
								</li>
							</ul>
						</div>
					</section>
				</footer>
				
								

				<!-- Copyright -->
					<div id="copyright">
						<ul><li>&copy; Untitled</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li></ul>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>